azure.gemspec

```diff
@@ -36,7 +36,11 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency('faraday',                 '~> 0.9')
   s.add_runtime_dependency('faraday_middleware',      '~> 0.10')
   s.add_runtime_dependency('mime-types',              ['>= 1', '< 4.0'])  # vagrant-share and other stuff relies on 1
-  s.add_runtime_dependency('nokogiri',                '~> 1.6')
+  if RUBY_VERSION.match(/(^1\.9.*)|(^2\.0.*)/)
```

Comment:
why use regex? Cann't we use `>` checks?

Reference: https://github.com/search?utf8=%E2%9C%93&q=RUBY_VERSION+language%3ARuby+extension%3A.gemspec&type=Code&ref=advsearch&l=Ruby&l=